README.md

```diff
@@ -1 +1,45 @@
-# WM-Website-Development
\ No newline at end of file
+# WM-Website-Development
+
+> This is capstone one of module one. 
+# Screenshot's 
+![screenshots-capstone-1](https://user-images.githubusercontent.com/69077061/157525702-966cc141-e7a1-4a41-99e1-d31efcfcaf57.png)
+
+## Built With
+
+- HTML
+- CSS 
+- JavaScript
+
+## Live Demo
+
+[Go live](https://tadesse-alemayehu.github.io/LEARN-TO-CODE/) 🙂
```

Comment:
A link to the deployed project is present in the README 📝

Reply:
@mateo951 Thank you.